services/web/server/src/simcore_service_webserver/announcements/_redis.py

```diff
@@ -0,0 +1,32 @@
+""" Repository layer using redis
+"""
+
+import logging
+
+import redis.asyncio as aioredis
+from aiohttp import web
+from pydantic import ValidationError
+
+from ..redis import get_redis_announcements_client
+from ._models import Announcement
+
+_logger = logging.getLogger(__name__)
+
+
+async def list_announcements(app: web.Application) -> list[Announcement]:
+    redis_client: aioredis.Redis = get_redis_announcements_client(app)
+    published = await redis_client.get(name="announcements") or []
+    announcements = []
+    for i, item in enumerate(published):
+        try:
+            announcements.append(Announcement.parse_raw(item))
+        except ValidationError:  # noqa: PERF203
+            #
+            # Announcements are manually stored in redis
+            # w/o guarantees. We validate them here and skip
+            # if fail.
+            _logger.exception(
+                "Invalid announcement #%d published in redis. Skipping. [=%s]", i, item
```

Comment:
We could alert based on this logline, but i guess it might be overkill for this feature

Reply:
this is only a precaution to point out that the "publisher" wrote something wrong. 
As I wrote in the comment above, this first version we  need to write in redis-commander "by hand" the announcment . 

As soon as we have a validated entrypoint to set the value, this part will dissapear